code/datums/mind.dm

```diff
@@ -473,8 +473,8 @@ datum/mind
 	else
 		mind = new /datum/mind(key)
 		mind.original = src
-		if(ticker) ticker.minds += mind
-		else log_world("## DEBUG: mind_initialize(): No ticker ready yet! Please inform Carn")
+		if(SSticker) SSticker.minds += mind
+		else log_world("## DEBUG: mind_initialize(): No SSticker ready yet! Please inform Carn")
```

Comment:
Carn has left the project since 5 years, may want to update the log.

Reply:
Carn~~